pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -499,6 +500,15 @@ func testHTTPSClassification(t *testing.T, tr *Tracer, clientHost, targetHost, s
 		},
 	}
 
+	// makeRequest is a helper that makes a GET request and handle the response.
+	makeRequest := func(t require.TestingT, client *nethttp.Client, url string) {
+		r, err := client.Get(url)
+		require.NoError(t, err)
```

Comment:
using `require` here is wrong, and will result in running `makeRequest` only once.
`require.NoError` wraps `assert.NoError` but in case of a failure, it calls `t.FailNow` which halts the test.